### PR TITLE
Added Groups Component;

### PR DESCRIPTION
**Pull request by ipykalo**
Added Groups Service;
Added Tooltip Directive;
Added User Module
Added User-info Service;
Implemented Logout logic;
Implemented success login logic;

### CODE REVIEW

File: src/app/modules/share/header/header.component.ts [hunk 1]
```diff
@@ -1,39 +1,101 @@
...
 @Component({
   selector: 'ls-header',
   templateUrl: './header.component.html',
   styleUrls: ['./header.component.scss']
 })
 export class HeaderComponent implements OnInit, OnDestroy {
-  private subscription: Subscription;
+  private navSubscription: Subscription;
+  private authSubscription: Subject<any> = new Subject();
+  private userInfoSubscription: Subject<any> = new Subject();
```
Comment (denis-starchenko) [inline src/app/modules/share/header/header.component.ts]: U can use one subject for unsubscribe from subscriptions in couple with takeUntil operator.

For example: `destroy$ = new Subject();`

File: src/app/modules/share/header/user-info.service.ts [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import { Injectable } from '@angular/core';
+import { HttpClient } from "@angular/common/http";
+import { environment } from "../../../../environments/environment";
```
Comment (denis-starchenko) [inline src/app/modules/share/header/user-info.service.ts]: U can add short path for this case. Please see example https://medium.com/rpdstartup/simplifying-typescript-import-paths-in-angular-c3249a4edbb3

File: src/app/modules/users/groups/groups.component.scss [hunk 3]
```diff
@@ -0,0 +1,26 @@
+@import '../../../../scss/constants/colors';
```
Comment (denis-starchenko) [inline src/app/modules/users/groups/groups.component.scss]: U can use short name for import. Please see https://scotch.io/tutorials/angular-shortcut-to-importing-styles-files-in-components

File: src/app/modules/share/header/header.component.ts (hunk 1 above)
Comment (ipykalo) [reply src/app/modules/share/header/header.component.ts]: Fixed

File: src/app/modules/share/header/user-info.service.ts (hunk 2 above)
Comment (ipykalo) [reply src/app/modules/share/header/user-info.service.ts]: Fixed

File: src/app/modules/users/groups/groups.component.scss (hunk 3 above)
Comment (ipykalo) [reply src/app/modules/users/groups/groups.component.scss]: Fixed

**Pull request merged**